apis/tempo/v1alpha1/microservices_types.go

```diff
@@ -367,24 +369,6 @@ type MicroservicesList struct {
 	Items           []Microservices `json:"items"`
 }
 
-// ImagesSpec defines the image for each container.
-type ImagesSpec struct {
```

Comment:
Why did you need to change the location of this for creating the Ingress/OpenShift Routes?

Reply:
Currently `projectconfig_types.go` (`apis/config/v1alpha1`) has a dependency on `microservices_types.go` (`apis/tempo/v1alpha1`) to use `ImagesSpec`. For this PR I need access to the projectconfig in the webhook, so I need to import `apis/config/v1alpha1` from `apis/tempo/v1alpha1`, and then we have a circular dependency. Therefore I moved it to `apis/config/v1alpha1`.